### DISCUSSION TITLE
Schema Editor - Change order of zoom icons

### DISCUSSION THREAD

**Discussion by haleemhosein** [TiagoSilvaPereira/vemto-issues#255] [Ideas & Suggestions]
The zoom out icon (-) should be on the left and the zoom in icon (+) should be on the right, similar to how it's shown in a web browser.

![image](https://user-images.githubusercontent.com/59444383/140654524-551a4aa3-8b51-4fb8-8920-eec862b81a9d.png)

![image](https://user-images.githubusercontent.com/59444383/140654542-93372139-bd8c-4e0c-b804-49452d7b2265.png)

**Comment by TiagoSilvaPereira** [comment]
@haleemhosein thank you, we changed it after the 1.0.0 version
